commands/keplr.js

```diff
@@ -214,6 +229,17 @@ const keplr = {
     return true;
   },
 
+  async getWalletAddress() {
+    await playwright.switchToKeplrWindow();
+    await module.exports.goToHome();
+    const page = await playwright.keplrWindow();
+    await playwright.waitAndClickByText(notificationPageElements.copyAddress);
+    await page.click(notificationPageElements.copyWalletAddressSelector);
+    const walletAddress = clipboardy.readSync();
+    await playwright.switchToCypressWindow();
```

Comment:
I've initially tried to get value from the clipboard by executing a script through `execute` in Playwright. When I tried to get value from the clipboard in the browser, it asked me for permission to access it. It was showing a popup. I tried these things which didn't work:

- I tried changing permissions in the browser config but they didn't apply. With the Playwright, it involved setting up a whole new context. With Cypress, the options were not according to our needs. 
- I also tried using the `Tab` key. I tried to press the `Tab` key 3 times using Playwright, to focus on the permission popup and then press the `Enter` key to get the permission but encountered issues related to focus there.  

I tried a few more things which mainly involved executing a script through `execute`. But didn't work. So I explored some libraries and found `clipboardy`. I feel it is an intuitive way of catching the value from the clipboard without adding any complexity. And it is working fine.